inventory.ini

```diff
@@ -0,0 +1,4 @@
+18.169.140.162
```

Comment:
Nothing wrong with this, but we don't typically list the control node in the inventory (unless you might want to target it)